docs/cwl/conformance.rst

```diff
@@ -0,0 +1,93 @@
+.. _cwlConformance:
+
+Toil CWL Conformance v1.2.1
```

Comment:
This sounds like this is v1.2.1 of the conformance information. We probably can leave out the version.
```suggestion
Toil CWL Conformance
```